### PR TITLE
Continuing #63

### PR DESCRIPTION
**Pull request by mih**

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -318,9 +318,11 @@ The [datalad-installer](https://github.com/datalad/datalad-installer) (also avai
 
 DataLad has been developed openly in a public repository ([github.com/datalad/datalad](https://github.com/datalad/datalad)) since its inception in 2013.
 At the time of this publication, the repository amassed over 13.5k commits, 2.5k merged PRs, and 2.3k closed (+700 open) issues from over 30 contributors.
-Issue tracker, labels, milestones, and pull requests (from personal forks) are used to coordinate development.
-DataLad heavily relies on the versatility and stability of the underlying core tools---Git and git-annex.
-To avoid reimplementing the wheel and to benefit the git-annex user community at large, many aspects of the desired functionality are and have been implemented directly in git-annex through collaboration with the git-annex developer Joey Hess [@git-annex:projects-datalad].
+Issue tracker, labels, milestones, and pull requests are used to coordinate development.
+The development process of DataLad is not isolated from its fundational building blocks.
```
Comment (yarikoptic) [inline paper.md]:
```suggestion
The development process of DataLad is not isolated from its foundational building blocks.
```

File: paper.md [hunk 2]
```diff
@@ -318,9 +318,11 @@ The [datalad-installer](https://github.com/datalad/datalad-installer) (also avai
...
+For every new feature or bug fix the most appropriate software layer is determined to maximize the size of the benefitting user base, but, importantly, also the associated developer audience.
```
Comment (yarikoptic) [inline paper.md]:
```suggestion
For every new feature or bug fix the most appropriate software layer is determined to maximize the size not only of the benefitting user base, but, importantly, also of the associated developer audience.
```

**Comment by adswa** [comment]
I like the changes in this PR

**Review by yarikoptic — COMMENTED** [review]
Few small fixes

File: paper.md (hunk 2 above)
Comment (mih) [reply paper.md]: FTR: I don't like the "not only" construct.

Comment (adswa) [reply paper.md]: ehm, I think this isn't a proper sentence anymore with this change... My English fails me, no placement of "not only" seems to make sense without further word additions...  But I like the sentence without the additions actually.

Comment (bpoldrack) [reply paper.md]: Reads correct to me, but adds nothing. So, I'm with Michael here.

Comment (yarikoptic) [reply paper.md]: May be 
```suggestion
For every new feature or bug fix the most appropriate software layer is determined to maximize the size of the benefitting user base, and, importantly, also the associated developer audience.
```
then? To my ear but is sticking out there alone. @kyleam wdyt?

Comment (kyleam) [reply paper.md]: I trip a bit on the "but" too and think "and" (your last suggestion) reads better.

A related issue I have when parsing this sentence is expecting more to come after "developer audience" due to the comma before "but" (or "and" in your version).  To my eyes, it'd be clearer without the comma (i.e. "user base and, importantly, ...").

Comment (mih) [reply paper.md]: Sounds good
```suggestion
For every new feature or bug fix the most appropriate software layer is determined to maximize the size of the benefitting user base and, importantly, also the associated developer audience.
```

**Pull request merged**